<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-22T12:04:28 IST" name="Suite" finished-at="2024-03-22T12:05:06 IST" duration-ms="38294">
    <groups>
    </groups>
    <test started-at="2024-03-22T12:04:28 IST" name="Test" finished-at="2024-03-22T12:05:06 IST" duration-ms="38294">
      <class name="TestClasses.TestClass1">
        <test-method is-config="true" signature="initialiseDriver()[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:04:29 IST" name="initialiseDriver" finished-at="2024-03-22T12:04:41 IST" duration-ms="12145" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialiseDriver -->
        <test-method signature="checkTest(java.lang.String,java.lang.String)[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:04:41 IST" name="checkTest" data-provider="provideData" finished-at="2024-03-22T12:04:41 IST" duration-ms="280" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[dummy@maveric.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Maveric@1234]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at TestClasses.TestClass1.checkTest(TestClass1.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:04:41 IST" name="afterMethod" finished-at="2024-03-22T12:04:47 IST" duration-ms="5942" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="initialiseDriver()[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:04:47 IST" name="initialiseDriver" finished-at="2024-03-22T12:05:00 IST" duration-ms="12527" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialiseDriver -->
        <test-method signature="checkTest(java.lang.String,java.lang.String)[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:05:00 IST" name="checkTest" data-provider="provideData" finished-at="2024-03-22T12:05:00 IST" duration-ms="610" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[maveric@maveric.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Citinam@nam12]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at TestClasses.TestClass1.checkTest(TestClass1.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestClasses.TestClass1@d554c5f]" started-at="2024-03-22T12:05:00 IST" name="afterMethod" finished-at="2024-03-22T12:05:06 IST" duration-ms="5951" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestClasses.TestClass1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
